gnomad_constraint/pipeline/constraint_pipeline.py

```diff
@@ -289,6 +346,39 @@ def main(args):
         ),
         action="store_true",
     )
+    parser.add_argument(
+        "--apply-models",
+        help=(
+            "Apply plateau and coverage models to variants in exome sites Table and"
+            " context Table to compute expected variant counts."
+        ),
+        action="store_true",
+    )
+    parser.add_argument(
+        "--partition-hint-when-counting-variants",
```

Comment:
Maybe:

```suggestion
        "--apply-obs-pos-count-partition-hint",
```

Reply:
I felt 'apply' may imply that it's only used in `apply_models`, so I change it to `count-obs-pos-vars-partition-hint`!